olad/ClientBroker.cpp

```diff
@@ -54,6 +54,21 @@ void ClientBroker::SendRDMRequest(const Client *client,
                         callback));
 }
 
+void ClientBroker::RunRDMDiscovery(const Client *client,
+                                   Universe *universe,
+                                   bool full_discovery,
+                                   ola::rdm::RDMDiscoveryCallback *callback) {
+  if (!STLContains(m_clients, client)) {
+    OLA_WARN << "Running RDM discovery but the client doesn't exist in the "
+             << "broker!";
+  }
```

Comment:
It's not a bug on the client side, so knowing which client it is isn't really going to help.